src/core/dxf/qgsdxfexport.cpp

```diff
@@ -150,7 +150,16 @@ void QgsDxfExport::writeGroup( const QColor &color, int exactMatchCode, int rgbC
 
   int c = ( color.red() & 0xff ) * 0x10000 + ( color.green() & 0xff ) * 0x100 + ( color.blue() & 0xff );
   writeGroup( rgbCode, c );
-  if ( transparencyCode != -1 && color.alpha() < 255 )
+
+  if ( transparencyCode > 0 )
+    return;
```

Comment:
Does this check do anything? It's the end of the method so the return is ahead anyway

Reply:
> Does this check do anything? It's the end of the method so the return is ahead anyway

You are right, thanks. I deleted these rows. BTW, can you help me to resolve this error:
*** SIP file not up to date: core/auto_generated/dxf/qgsdxfexport.sip

Reply:
You need to rebase or merge master into your branch.